sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/AlreadyExistException.scala

```diff
@@ -31,7 +31,7 @@ class TableAlreadyExistsException(db: String, table: String)
   extends AnalysisException(s"Table or view '$table' already exists in database '$db'")
 
 class TempTableAlreadyExistsException(table: String)
```

Comment:
shall we rename it to `TempViewAlreadyExistsException`?

Reply:
I think we should rename it. But @gatorsmile  said "We do not want to introduce a new exception type. In contrast, we planned to remove all these exception sub-types because PySpark might output a confusing error message.".  So I revert it.